app/assets/javascripts/plugins/analyticsPlugin.js

```diff
@@ -7,18 +7,16 @@ import { getManifest, getCompanionWindow } from 'mirador/dist/es/src/state/selec
 function* onSetCanvas({ type, windowId, canvasId }) {
   const { id: manifestId } = yield select(getManifest, { windowId });
   yield put({ type: 'mirador/analytics', payload: {
-    hitType: 'event',
-    eventCategory: manifestId,
-    eventAction: type,
-    eventLabel: canvasId,
+    event_category: manifestId,
```

Comment:
not a huge concern, but aren't parameters in javascript typically camelcased (as they were before), why did you migrate away from this convention?

Reply:
This is the syntax used in all the official Google guides, e.g. https://developers.google.com/analytics/devguides/migration/ua/analyticsjs-to-gtagjs#measure_events

Reply:
I only see that in the parameters to the `gtag` call.  I don't think we need to change all the other parameters in this plugin.

Reply:
Ah I see what you mean. I can change that back, I guess keeping them uniform just was cleaner in my mind.

Reply:
🤷

Reply:
I'll change it